SignalServiceKit/src/Devices/OWSRecordTranscriptJob.m

```diff
@@ -86,10 +103,19 @@ - (void)runWithAttachmentHandler:(void (^)(TSAttachmentStream *attachmentStream)
         return;
     }
 
+    if (outgoingMessage.body.length < 1 && outgoingMessage.attachmentIds.count < 1) {
+        // TODO: Is this safe?
+        DDLogInfo(@"Ignoring message transcript for empty message.");
```

Comment:
Check this out. IMO we want to avoid creating a new outgoing message that has neither a body nor an attachment.

Reply:
Paranoid, I've never seen this.  Made it a `OWSFail()`.